ochre/utils/hpxml.py

```diff
@@ -100,22 +100,23 @@ def get_boundaries_by_zones(boundaries, default_int_zone='Indoor', default_ext_z
 
 def parse_hpxml_surface(bd_name, bd_data):
     out = {
-        'Area (m^2)': convert(bd_data['Area'], 'ft^2', 'm^2'),
         'Interior Zone': bd_data['Interior Zone'],
         'Exterior Zone': bd_data['Exterior Zone'],
+        'Area (m^2)': convert(bd_data['Area'], 'ft^2', 'm^2'),
     }
 
+    # Add azimuth if it exists
+    azimuth = bd_data.get('Azimuth')
+    if azimuth is not None:
+        out['Azimuth (deg)'] = azimuth
+
     # Add R value if it exists
     r_value = bd_data.get('Insulation', {}).get('AssemblyEffectiveRValue', bd_data.get('RValue'))
     if r_value:
         out['Boundary R Value'] = r_value
 
     # Exterior boundary properties (roofs, walls, and rim joists). Not used for raised floors
     if bd_data.get('Exterior Zone') == 'Outdoor':
-        # TODO: get azimuth from Orientation, get absorptivity from color
```

Comment:
Did we mean to lose this TODO? Absorptivity from color still seems like it'd be nice...

Reply:
yes, it's old. We already get azimuth and absorptivity directly from the HPXML.